### ISSUE TITLE
[ENH]: Improve NNs layers

### ISSUE THREAD

**Issue by ishii-norimi** [ai-on-browser/ai-on-browser.github.io#651]
### Suggestion feature

- Add `argmin`, `softmin` and `logsoftmax` layers
- Add ONNX support for `argmax`, `batch_normalization`, `clip` and `softmax` layers

### Is this a feature to be added to the GUI or to the package?

package
